controllers/auth.js

```diff
@@ -36,6 +36,7 @@ auth.mount = app => {
    * @api {post} /auth Authenticate a user
    * @apiName Authenticate
    * @apiGroup Authentication
+   * @apiVersion 2.0.0
```

Comment:
Is there a way to have a global api version instead of changing it on every endpoint every time the version changes?

Reply:
I looked and couldn't find anything. I did want to experiment with only providing the version on the old endpoints - maybe, if no version is provided, apiDoc will default to the current version. I'll try that, and if it doesn't work, we'll be stuck with how it currently is.